code/datums/martial/cqm.dm

```diff
@@ -0,0 +1,140 @@
+		D.emote("spin")
+		if(D.confused <= 20)
+			D.confused = CLAMP(D.confused + 10, 0, 20)
+		A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+		playsound(get_turf(D), 'sound/weapons/thudswoosh.ogg', 30, 1, -1)
+		return TRUE
+
+	return basic_hit(A,D)
+
+//Tounge Pull, Deal 10 brute to the head(reduced by armor(space magic), Deals damage to the targets tounge and restricts speech for a bit.
+/datum/martial_art/cqm/proc/toungePull(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	var/def_check = D.getarmor(BODY_ZONE_HEAD, "melee")
+	log_combat(A, D, "Tounge Pulled (Close Quarters Mimery)")
+	D.visible_message("<span class='warning'>[A] pulls [D]'s tounge painfully!</span>", \
+		"<span class='userdanger'>[A] pulls your tounge painfully restricting your speech!</span>")
+	D.apply_damage(10, A.dna.species.attack_type, BODY_ZONE_HEAD, def_check)
+	D.adjustOrganLoss(ORGAN_SLOT_TONGUE, 15, 200)
+	D.Jitter(20)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	playsound(get_turf(D), 'sound/effects/hit_punch.ogg', 30, 1, -1)
+	if(D.silent <= 10)
+		D.silent = CLAMP(D.silent + 10, 0, 10)
+	return TRUE
+
+//Throat Punch, Prevents breating for a moment, deals oxygen damage and restricts speech for a some time.
+/datum/martial_art/cqm/proc/throatPunch(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	log_combat(A, D, "Throat punched (Close Quarters Mimery)")
+	D.visible_message("<span class='warning'>[A] Punches [D]'s throat!</span>", \
+		"<span class='userdanger'>[A] punches your throat restricting your speech and breathing!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	D.losebreath += 3
+	D.adjustOxyLoss(10)
+	if(D.silent <= 15)
+		D.silent = CLAMP(D.silent + 15, 0, 15)
+	return TRUE
+
+//Arm Pull, a weaker wrist wrench that wont grant distance from batons. Disarms and very briefly stuns the target for three seconds as well as dealing 5 brute to either arm.
+/datum/martial_art/cqm/proc/armPull(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!D.stat && !D.IsStun() && !D.IsParalyzed())
+		log_combat(A, D, "Arm pulled (Close Quarters Mimery)")
+		D.visible_message("<span class='warning'>[A] grabs [D]'s arm and pulls it sideways painfully!</span>", \
+			"<span class='userdanger'>[A] grabs and pulls your arm painfully to the side!</span>")
+		playsound(get_turf(A), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+		A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+		D.apply_damage(5, BRUTE, pick(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM))
+		D.dropItemToGround(D.get_active_held_item())
+		D.emote("scream")
+		D.Stun(30)
+		return TRUE
+
+	return basic_hit(A,D)
+
+//Mime special, RIP HIS TOUNGE OUT AND CRUSH IT BEFORE HIS EYES! 25 brute to the head and a 6s stun also destroying the targets tounge and making them BLEED!
+/datum/martial_art/cqm/proc/mimeSpecial(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	var/obj/item/organ/tongue/T = D.getorganslot(ORGAN_SLOT_TONGUE)
```

Comment:
if(T) check should be way above. Whole move doesn't make sense if a person doesn't have tongue in first place.